include/odp/api/spec/queue.h

```diff
@@ -50,7 +50,9 @@ odp_queue_t odp_queue_create(const char *name, const odp_queue_param_t *param);
  * Destroy ODP queue
  *
  * Destroys ODP queue. The queue must be empty and detached from other
- * ODP API (crypto, pktio, etc). Application must ensure that no other
+ * ODP API (crypto, pktio, etc). The queue must not be in use in any CoS. If it
+ * is, assign another queue to the CoS, or destroy the CoS, before calling this
+ * function. Application must ensure that no other
```

Comment:
The added text is good, but isn't it already said there ("The queue must be empty and detached from other ODP API (crypto, pktio, etc)."? If clarification is needed, maybe it is needed for other cases that classifier too?

Reply:
Answer to both questions is yes, probably. This change could just be dropped. But I still kept it, anyway, in v2.

Reply:
Similarly to pool API, queue API should not give too specific instructions about classifier API, but just remind that any usage of a destroyed queue leads to problems.

Classifier and timer could be added to the list of "other APIs", as those are common users of destination queues.